doc/topics/development/contributing.rst

```diff
@@ -219,201 +219,34 @@ Fork a Repo Guide_>`_ and is well worth reading.
-
-Prior to tagging an official release, a branch will be created when the SaltStack
-release team is ready to tag. The dot release branch is created from a main release
-branch. The dot release branch will be the same name as the tag minus the ``v``.
-For example, the ``2017.7.1`` dot release branch was created from the ``2017.7``
-main release branch. The ``v2017.7.1`` release was tagged at the ``HEAD`` of the
-``2017.7.1`` branch.
-
-This branching strategy will allow for more stability when there is a need for
-a re-tag during the testing phase of the release process and further increases
-stability.
-
-Once the dot release branch is created, the fixes required for a given release,
-as determined by the SaltStack release team, will be added to this branch. All
-commits in this branch will be merged forward into the main release branch as
-well.
-
-Merge Forward Process
-=====================
-
-The Salt repository follows a "Merge Forward" policy. The merge-forward
-behavior means that changes submitted to older main release branches will
-automatically be "merged-forward" into the newer branches.
-
-For example, a pull request is merged into ``2017.7``. Then, the entire
-``2017.7`` branch is merged-forward into the ``2018.3`` branch, and the
-``2018.3`` branch is merged-forward into the ``|repo_primary_branch|`` branch.
-
-This process makes is easy for contributors to make only one pull-request
-against an older branch, but allows the change to propagate to all **main**
-release branches.
-
-The merge-forward work-flow applies to all main release branches and the
-operation runs continuously.
-
-Merge-Forwards for Dot Release Branches
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The merge-forward policy applies to dot release branches as well, but has a
-slightly different behavior. If a change is submitted to a **dot** release
-branch, the dot release branch will be merged into its parent **main**
-release branch.
-
-For example, a pull request is merged into the ``2017.7.2`` release branch.
-Then, the entire ``2017.7.2`` branch is merged-forward into the ``2017.7``
-branch. From there, the merge forward process continues as normal.
-
-The only way in which dot release branches differ from main release branches
-in regard to merge-forwards, is that once a dot release branch is created
-from the main release branch, the dot release branch does not receive merge
-forwards.
-
-.. note::
+When the time comes for a new release which should be every 3 to 4 months
+the release will be cut from the master.
 
-    The merge forward process for dot release branches is one-way:
-    dot release branch --> main release branch.
+To be able to merge code. The code must have well written test.
+Please note you are only expected to write test for what you did not the whole modules or function.
+All tests must also pass.
+The salt stack employee that reviews your pull request might
+request changes or deny the pull request for various reasons.
```

Comment:
Let's maybe put this into bullet points for more clarify with the bullet headings in bold. I'm also fixing a few minor grammar errors or some phrasing suggestions. Here's my fix:

To be able to merge code:

1. **The code must have a well-written test.** Note that you are only expected to write tests for what you did, not the whole modules or function.
2. **All tests must pass.**

The SaltStack employee that reviews your pull request might request changes or deny the pull request for various reasons.